spec/cc/cli/command_spec.rb

```diff
@@ -5,7 +5,7 @@ module CC::CLI
     describe "#require_codeclimate_yml" do
       it "exits if the file doesn't exist" do
         Dir.chdir(Dir.mktmpdir) do
-          stdoit, stderr = capture_io do
+          stdout, stderr = capture_io do
```

Comment:
If this variable is unused (which is likely b/c you renamed it and nothing broke), please call it `_` to convey that.